dotCMS/src/main/java/com/dotcms/ai/config/AiModelConfig.java

```diff
@@ -0,0 +1,47 @@
+package com.dotcms.ai.config;
+
+import java.util.Map;
+
+/**
+ * Encapsulates the Configuration for a model
+ * @author jsanca
+ */
+public class AiModelConfig {
+
+    public static final String API_KEY = "key";
+    public static final String VENDOR  = "vendor";
+    public static final String MODEL   = "model";
+    public static final String API_URL = "apiUrl";
+
+    private final String name;
```

Comment:
Not that this code needs to be super high performance but there is a java Map called `EnumMap` that uses Enums as a key and is supposed to be much faster than a HashMap...  Nothing really to do here but I'm looking for a chance to use it :P